pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2867 @@
+	if m != nil {
+		return m.RepoTags
+	}
+	return nil
+}
+
+func (m *Image) GetRepoDigests() []string {
+	if m != nil {
+		return m.RepoDigests
+	}
+	return nil
+}
+
+func (m *Image) GetSize_() uint64 {
+	if m != nil && m.Size_ != nil {
+		return *m.Size_
+	}
+	return 0
+}
+
+type ListImagesResponse struct {
+	// List of images
+	Images           []*Image `protobuf:"bytes,1,rep,name=images" json:"images,omitempty"`
+	XXX_unrecognized []byte   `json:"-"`
+}
+
+func (m *ListImagesResponse) Reset()         { *m = ListImagesResponse{} }
+func (m *ListImagesResponse) String() string { return proto.CompactTextString(m) }
+func (*ListImagesResponse) ProtoMessage()    {}
+
+func (m *ListImagesResponse) GetImages() []*Image {
+	if m != nil {
+		return m.Images
+	}
+	return nil
+}
+
+type ImageStatusRequest struct {
+	// The spec of the image
+	Image            *ImageSpec `protobuf:"bytes,1,opt,name=image" json:"image,omitempty"`
+	XXX_unrecognized []byte     `json:"-"`
+}
+
+func (m *ImageStatusRequest) Reset()         { *m = ImageStatusRequest{} }
+func (m *ImageStatusRequest) String() string { return proto.CompactTextString(m) }
+func (*ImageStatusRequest) ProtoMessage()    {}
+
+func (m *ImageStatusRequest) GetImage() *ImageSpec {
+	if m != nil {
+		return m.Image
+	}
+	return nil
+}
+
+type ImageStatusResponse struct {
+	// The status of the image
+	Image            *Image `protobuf:"bytes,1,opt,name=image" json:"image,omitempty"`
+	XXX_unrecognized []byte `json:"-"`
+}
+
+func (m *ImageStatusResponse) Reset()         { *m = ImageStatusResponse{} }
+func (m *ImageStatusResponse) String() string { return proto.CompactTextString(m) }
+func (*ImageStatusResponse) ProtoMessage()    {}
+
+func (m *ImageStatusResponse) GetImage() *Image {
+	if m != nil {
+		return m.Image
+	}
+	return nil
+}
+
+// AuthConfig contains authorization information for connecting to a registry.
+type AuthConfig struct {
+	Username      *string `protobuf:"bytes,1,opt,name=username" json:"username,omitempty"`
+	Password      *string `protobuf:"bytes,2,opt,name=password" json:"password,omitempty"`
+	Auth          *string `protobuf:"bytes,3,opt,name=auth" json:"auth,omitempty"`
+	ServerAddress *string `protobuf:"bytes,4,opt,name=server_address" json:"server_address,omitempty"`
+	// IdentityToken is used to authenticate the user and get
+	// an access token for the registry.
+	IdentityToken *string `protobuf:"bytes,5,opt,name=identity_token" json:"identity_token,omitempty"`
+	// RegistryToken is a bearer token to be sent to a registry
+	RegistryToken    *string `protobuf:"bytes,6,opt,name=registry_token" json:"registry_token,omitempty"`
+	XXX_unrecognized []byte  `json:"-"`
+}
+
+func (m *AuthConfig) Reset()         { *m = AuthConfig{} }
+func (m *AuthConfig) String() string { return proto.CompactTextString(m) }
+func (*AuthConfig) ProtoMessage()    {}
+
+func (m *AuthConfig) GetUsername() string {
+	if m != nil && m.Username != nil {
+		return *m.Username
+	}
+	return ""
+}
+
+fun
```

Comment:
The function should have another parameter, `interceptor grpc.UnaryServerInterceptor`, to match `grpc.MethodDesc.Handler`.